gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -727,6 +727,8 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
   if (filter->mode == GTT_UNKNOWN || filter->option == NULL)
     return;
 
+  gchar *filter_name = gst_object_get_name ((GstObject *) filter);
```

Comment:
declare local val first. (prevent build warning)
```
gchar *filter_name;

filter_name = gst_object_get_name ((GstObject *) filter);
```

Reply:
You're right. I will.